src/app/app-routing.module.ts

```diff
@@ -11,6 +11,10 @@ const routes: Routes = [
     path: 'project-details',
     loadChildren: () => import('./project/project.module').then( m => m.ProjectPageModule),
   },
+  {
```

Comment:
Please delete this SaveModule pages and its files, because we will be using a generic listing page here